Traceback (most recent call last):
  File "/home/Capstone_Design_Troy/judge/src/web/add_problem/make_guideline.py", line 41, in <module>
    part = chunk.decode('utf-8')
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xeb in position 4095: unexpected end of data
Traceback (most recent call last):
  File "/home/Capstone_Design_Troy/judge/src/web/add_problem/make_guideline.py", line 38, in <module>
    chunk = s.recv(buffer_size)
ConnectionResetError: [Errno 104] Connection reset by peer
